servicedirectory/snippets_test.py

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env python
+
+# Copyright 2020 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from os import environ
+import uuid
+
+from google.cloud import servicedirectory_v1beta1
+
+import snippets
+
+PROJECT_ID = environ['GOOGLE_CLOUD_PROJECT']
+LOCATION_ID = 'us-east1'
+NAMESPACE_ID = f'test-namespace-{uuid.uuid4().hex}'
+SERVICE_ID = f'test-service-{uuid.uuid4().hex}'
+ENDPOINT_ID = f'test-endpoint-{uuid.uuid4().hex}'
+ADDRESS = '1.2.3.4'
+PORT = 443
+
+
+def teardown_module():
+    client = servicedirectory_v1beta1.RegistrationServiceClient()
+    response = client.list_namespaces(
+        parent=f'projects/{PROJECT_ID}/locations/{LOCATION_ID}')
+    for namespace in response.namespaces:
```

Comment:
@rebeccawshaw 

This code is deleting all the namespaces. This will cause test failures when we run the tests simultaneously. Can you only delete the namespace used in the test?

Reply:
@tmatsuo ah done.